qemu/tests/iperf_test.py

```diff
@@ -91,10 +91,8 @@ def rss_test():
         rss_test_cmd = utils_misc.set_winutils_letter(
             guest_session, params["rss_test_cmd"])
         output = guest_session.cmd_output(rss_test_cmd)
-        rss_statistics = output.splitlines()[1].split()
-        rxerrors = int(rss_statistics[-4])
-        rxmissed = int(rss_statistics[-2])
-        if not (rxerrors == 0 and rxmissed == 0):
+        rss_statistics = output.strip().split('\n')
+        if "Errors=0" and "Misses=0" not in rss_statistics:
```

Comment:
bool(“Errors=0”) always equal to `True`, I would suggest using `re` module.

Reply:
> bool(“Errors=0”) always equal to `True`, I would suggest using `re` module.

So you mean this line should be change to the following format, right?
`if re.search("rss_statistics", "Errors=0") and re.search("rss_statistics", "Misses=0"):`

Reply:
You are comparing strings, this is unreasonable. Please check how to use `re` module to match multiple pattern.

Reply:
> You are comparing strings, this is unreasonable. Please check how to use `re` module to match multiple pattern.

Ok, got it your point, I will update it ASAP.

Reply:
@PaulYuuu Please help me review it again, thanks a lot.